docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+
+services:
+  db:
+    image: postgres
+    container_name: postgres
+    restart: always
+    ports:
+      - "6500:5432"
```

Comment:
Nice job with figuring out docker compose! Just curious why did you assigned 6500

Reply:
It's what some of the tutorials said to do 😅  some also said 5432:5432, is that better?